packages/components/src/DateTimePickers/DateTimePicker/DateTimePicker.component.js

```diff
@@ -13,34 +13,44 @@ class DateTimePicker extends React.Component {
 
 		this.state = {
 			isDateTimeView: true,
-			currentCalendar: {
+			calendar: {
 				monthIndex: now.getMonth(),
 				year: now.getFullYear(),
 			},
+			selectedDate: undefined,
 		};
 
 		this.setDateTimeView = this.setView.bind(this, true);
 		this.setMonthYearView = this.setView.bind(this, false);
-		this.onSelectMonth = this.onSelectMonth.bind(this);
-		this.onSelectYear = this.onSelectYear.bind(this);
-		this.onSelectMonthYear = this.onSelectMonthYear.bind(this);
+		this.onSelectCalendarMonth = this.onSelectCalendarMonth.bind(this);
+		this.onSelectCalendarYear = this.onSelectCalendarYear.bind(this);
+		this.onSelectCalendarMonthYear = this.onSelectCalendarMonthYear.bind(this);
+		this.onSelectDate = this.onSelectDate.bind(this);
 	}
 
-	onSelectMonthYear(newCalendar) {
+	onSelectDate(selectedDate) {
+		this.setState({ selectedDate });
+	}
+
+	onSelectCalendarMonthYear(newCalendar) {
 		this.setState(previousState => ({
-			currentCalendar: {
-				...previousState.currentCalendar,
+			calendar: {
+				...previousState.calendar,
 				...newCalendar,
 			},
 		}));
 	}
 
-	onSelectMonth(monthIndex) {
-		this.onSelectMonthYear({ monthIndex });
+	onSelectCalendarMonth(calendarMonthIndex) {
+		this.onSelectCalendarMonthYear({
+			monthIndex: calendarMonthIndex,
```

Comment:
please rename calendarMonthIndex -> index
by the way index is 0->11 ?
if 1_>12 it s not an index is the the month number

Reply:
it's really the index [0-11] like in Date API

Reply:
✔